dev-libs/svrcore/svrcore-4.1.2.ebuild

```diff
@@ -0,0 +1,34 @@
+# Copyright 1999-2016 Gentoo Foundation
+# Distributed under the terms of the GNU General Public License v2
+# $Id$
+
+EAPI=6
+
+inherit multilib autotools
```

Comment:
why do you inherit multilib? `EAPI=6` supports `get_libdir` without `multilib.eclass`.